src/lib/components/object-detail-item.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	import { onMount } from 'svelte'
+	import { slide } from 'svelte/transition'
+	import ButtonBlock from './button-block.svelte'
+	import Container from './container.svelte'
+	import Button from './button.svelte'
+	import ChevronLeft from './icons/chevron-left.svelte'
+	import Launch from './icons/launch.svelte'
+
+	let detailId: string
+
+	export let buttonLink: undefined | string = undefined
```

Comment:
```suggestion
	export let txHash: undefined | string = undefined
```
Let's name this `txHash` please

Reply:
> Let's name this `txHash` please

The reason I left the name more generic is because I figured in different objects, the button might be used for something other than the hash. But to be honest I'm not sure if that will ever happen 🤷‍♀️